easybuild/easyconfigs/e/EMBOSS/EMBOSS-6.6.0-intel-2017a-X11-20170314.eb

```diff
@@ -0,0 +1,52 @@
+# authors: Kenneth Hoste (Ghent University), George Tsouloupas <g.tsouloupas@cyi.ac.cy>, Fotis Georgatos <fotis@cern.ch>
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# http://hpcbios.readthedocs.org/en/latest/HPCBIOS_2012-94.html
+# foss-2016b modified by:
+# Adam Huffman
+# The Francis Crick Institute
+
+easyblock = 'ConfigureMake'
+
+name = 'EMBOSS'
+version = '6.6.0'
+x11ver = '20170314'
+versionsuffix = '-X11-%s' % x11ver
+
+homepage = 'http://emboss.sourceforge.net/'
+description = """EMBOSS is 'The European Molecular Biology Open Software Suite'.
+ EMBOSS is a free Open Source software analysis package specially developed for
+ the needs of the molecular biology (e.g. EMBnet) user community."""
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+
+source_urls = [
+    'ftp://emboss.open-bio.org/pub/EMBOSS/',
+    'ftp://emboss.open-bio.org/pub/EMBOSS/old/%(version_major_minor)s.0',
+]
+
+sources = [SOURCE_TAR_GZ]
+
+patches = ['EMBOSS_disable-embossupdate.patch']
+
+dependencies = [
+    ('X11', x11ver),
+    ('libgd', '2.2.4'),
+    ('libharu', '2.3.0'),
+    ('Java', '1.8.0_121', '', True),
```

Comment:
@hajgato consider using the most up to date `Java`?

Reply:
@verdurin Done!